book/intro.md

```diff
@@ -202,9 +204,9 @@ particular browser. Other people---including you---can and do contribute ideas,
 proposals, and implementations.
 
 What's amazing is that, despite the scale and the pace and the complexity, there
-is still plenty of room to contribute. Every browser today is open-source, which
+is still plenty of room to contribute. Every browser today is open source, which
 opens up its implementation to the whole community of web developers. Browsers
-evolve like giant R&D projects, where new ideas are constantly being proposed
+evolve like giant research projects, where new ideas are constantly being proposed
```

Comment:
The copyeditor just expanded this to "giant research and development projects" but I think the full phrase is unnecessary for the meaning.